www/taggame

```diff
@@ -66,6 +66,11 @@ for ($i = 0, $tags = array() ; ; $i++) {
 
 // make sure we process any persistent login state
 $userid = checkPersistentLogin();
+if (!$userid && $xml) {
+    $username = get_req_data("username");
+    $password = get_req_data("password");
+    list($userid, $errCode, $errMsg) = doLogin($db, $username, $password);
```

Comment:
If you're moving code, might as well change `list(x, y, z) =` to `[x, y, z] =`, which is I find cleaner, and is supported since PHP 7.1.